packages/fx-core/src/common/deps-checker/util/fileHelper.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @author Xiaofu Huang <xiaofhua@microsoft.com>
+ */
+import * as fs from "fs-extra";
+import * as path from "path";
+import { defaultHelpLink } from "../constant/helpLink";
+import { Messages } from "../constant/message";
+import { DepsCheckerError } from "../depsError";
+
+export async function unlinkSymlink(linkFilePath: string): Promise<void> {
+  try {
+    const stat = await fs.lstat(linkFilePath);
+    if (stat.isSymbolicLink()) {
+      await fs.unlink(linkFilePath);
+    }
+  } catch (error: unknown) {
+    const statError = error as { code?: string };
+    if (statError.code !== "ENOENT") {
+      throw error;
+    }
+  }
+}
+
+export async function createSymlink(target: string, linkFilePath: string): Promise<void> {
+  await unlinkSymlink(linkFilePath);
+  // check if destination already exists
+  if (await fs.pathExists(linkFilePath)) {
+    throw new DepsCheckerError(Messages.symlinkDirAlreadyExist(), defaultHelpLink);
+  }
+  await fs.mkdir(path.dirname(linkFilePath), { recursive: true, mode: 0o777 });
```

Comment:
Why `777` on directory?

Reply:
The default value is 777. 
![image](https://user-images.githubusercontent.com/49138419/233263989-2a068b8b-8a68-4dcd-a618-fe44ad855663.png)
I will remove the useless `mode`.